run-ghost.sh

```diff
@@ -1,54 +1,49 @@
 #!/bin/sh
-verbose='false'
-start='true'
-while getopts 'dv' flag; do
-  case "${flag}" in
-    d) start='false' ;;
-    v) verbose='true' ;;
-    *) start='true' ;;
-  esac
-done
-
-CONFIG="$GHOST_INSTALL/config.production.json"
-
-if [ -f $GHOST_INSTALL/config.override.json ]; then
-        echo "Ghost override provided. Override the internal configuration"
-        cp $GHOST_INSTALL/config.override.json $GHOST_INSTALL/config.production.json
-fi
+set -eo pipefail
 
-# Set Config
-if [ -z "$WEB_URL" ]; then
-	echo "WEB_URL is empty. Getting default: http://$(hostname -i):2368"
-	WEB_URL=http://$(hostname -i):2368
-fi
+if [[ "$*" == "node current/index.js" ]]; then 
+        CONFIG="$GHOST_INSTALL/config.production.json"
+
+        if [ -f $GHOST_INSTALL/config.override.json ]; then
+                echo "Ghost override provided. Override the internal configuration"
+                cp $GHOST_INSTALL/config.override.json $GHOST_INSTALL/config.production.json
+        fi
+
+        # Set Config
+        if [ -z "$WEB_URL" ]; then
+                echo "WEB_URL is empty. Getting default: http://$(hostname -i):2368"
+                WEB_URL=http://$(hostname -i):2368
+        fi
 
-echo "=> Change config based on ENV parameters:"
-echo "========================================================================"
-echo "      WEB_URL:        $WEB_URL"
-echo "========================================================================"
+        echo "=> Change config based on ENV parameters:"
+        echo "========================================================================"
+        echo "      WEB_URL:        $WEB_URL"
+        echo "========================================================================"
 
-sed -i "s|http://localhost:2368/|$WEB_URL|g" config.production.json
+        sed -i "s|http://localhost:2368/|$WEB_URL|g" config.production.json
 
-if [[ $verbose == 'true' ]]; then
-	cat $CONFIG
+        if [ -z "$(ls -A "$GHOST_CONTENT")" ]; then
+                echo "Missing content folder. Copying the default one..."
+                cp -r $GHOST_INSTALL/content.bck/* $GHOST_CONTENT
+        fi
 fi
 
-if [ -z "$(ls -A "$GHOST_CONTENT")" ]; then
-        echo "Missing content folder. Copying the default one..."
-        cp -r $GHOST_INSTALL/content.bck/* $GHOST_CONTENT
+if [[ "$*" == "init" ]]; then 
+        echo "Empty database. Initializing..."
+        knex-migrator init --mgpath "$GHOST_INSTALL/current"
+        exit 0
 fi
 
-if [ ! -s "$(awk '/"filename": "(.*)"/ {print $2}' $CONFIG | sed -e s/\"//g)" ]; then
+if [[ "$*" == "init start" ]]; then 
```

Comment:
As I said before, I would prefer to have the script taking care of this and not provide any attributes during runtime.